BSP/LED/led.c

```diff
@@ -78,9 +96,49 @@ void led_blinkTest(void)
         {
             HAL_GPIO_WritePin(cathodeLedIOtable[k].port, cathodeLedIOtable[k].pin, GPIO_PIN_RESET);
             HAL_GPIO_WritePin(anodeLedIOtable[a].port, anodeLedIOtable[a].pin, GPIO_PIN_SET);
-            HAL_Delay(10);
+            vTaskDelay(10);
             HAL_GPIO_WritePin(cathodeLedIOtable[k].port, cathodeLedIOtable[k].pin, GPIO_PIN_SET);
             HAL_GPIO_WritePin(anodeLedIOtable[a].port, anodeLedIOtable[a].pin, GPIO_PIN_RESET);
         }
     }
 }
+
+void led_squareTask(void *arg)
+{
+    static uint8_t alternateColLedOn;
```

Comment:
I know this is init to 0, but It does not hurt to write it down.
```suggestion
    static uint8_t alternateColLedOn = 0;
```
Seems to me that a bool is better suited here, this handle a toggle after all
```suggestion
    static bool isColLedOn = false;
```